pom.xml

```diff
@@ -285,6 +285,14 @@
                 <version>${open-metadata.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.odpi.egeria</groupId>
+                <artifactId>integration-daemon-services-api</artifactId>
+                <version>3.6-SNAPSHOT</version>
```

Comment:
Just a minor note if you happen to see this - could replace with ${open-metadata-version} to avoid so many literal versions